_field-types/mapping-parameters/ignore-above.md

```diff
@@ -0,0 +1,180 @@
+---
+layout: default
+title: ignore_above
+parent: Mapping parameters
+grand_parent: Mapping and field types
+nav_order: 45
+has_children: false
+has_toc: false
+---
+
+# ignore_above
+
+The `ignore_above` mapping parameter limits the maximum number of characters that are permitted for a string to be indexed. If the length of a string exceeds the specified threshold, the value is stored with the document but is not indexed. This can help prevent the index from bloating with unusually long values and can keep queries efficient.
```

Comment:
```suggestion
The `ignore_above` mapping parameter limits the maximum number of characters for an indexed string. If a string's length exceeds the specified threshold, the value is stored with the document but is not indexed. This can help prevent the index from bloating with unusually long values and can keep queries efficient.
```